code/__DEFINES/supermatter.dm

```diff
@@ -24,14 +26,16 @@
 #define HEALIUM_TRANSMIT_MODIFIER 2.4
 #define PROTO_NITRATE_TRANSMIT_MODIFIER 15
 #define ZAUKER_TRANSMIT_MODIFIER 20
+#define HELIUM_TRANSMIT_MODIFIER 5
+#define ANTINOBLIUM_TRANSMIT_MODIFIER 40
```

Comment:
brotherrrrrrr

Reply:
It's harder than tritium, so it might as well have a good reward. Would a power transmission of 40 break power balance?

Reply:
I'm not convinced there's a point to making harder tritium, why is that something we need?

Reply:
I use to be unsure on whether pure tritium engines were possible or not, and since it was basically the hardest gas, I got kinda disappointed when I found out how to run pure tritium engines, knowing there isn't really anything much harder to do.

I added the antinoblium interaction because I want to keep things fresh, and would be a fun challenge for anyone wanting to do something harder than tritium.